warmup.krun

```diff
@@ -141,9 +141,14 @@ if sys.platform.startswith("linux"):
         "sudo service cron stop",
         "sudo service atd stop",
         "sudo service postfix stop",
+        "sudo service networking stop",
     ]
 
+    PING_HOST = "192.168.1.245"
```

Comment:
Is it wise to have IP addresses hard-coded like this? I'm just thinking that if someone else tries to clone and run this repo....